RecastDemo/Source/main.cpp

```diff
@@ -686,15 +686,16 @@ int main(int /*argc*/, char** /*argv*/)
 				mouseOverMenu = true;
 			
 			int levelToLoad = -1;
-			for (int i = 0; i < files.size; ++i)
-			{
-				if (imguiItem(files.files[i]))
+			
+			for (unsigned int i = 0; i < files.size(); ++i) {
```

Comment:
If you really want to be pedantic, the value returned by files.size() is a `std::vector<std::string>::size_type` value, but who wants to write that? :P